src/widgets/mentors-wanted-course/ui/mentors-wanted-course.tsx

```diff
@@ -16,7 +16,7 @@ type MentorsWantedCourseProps = {
 export const MentorsWantedCourse = ({ link }: MentorsWantedCourseProps) => {
   return (
     <section className={cx('mentors-wanted', 'container')} data-testid="mentors-wanted">
-      <article className={cx('mentors-wanted-content', 'content')}>
+      <article className={cx('mentors-wanted-content', 'content', 'column-2')}>
```

Comment:
As I can see, we needn't here `mentors-wanted-content` class.

Reply:
Delete unused styles